packages/suite/src/components/wallet/Fees/Fees.tsx

```diff
@@ -157,7 +157,12 @@ export const Fees = <TFieldValues extends FormState>({
     const error = errors.selectedFee;
     const selectedLevel = feeInfo.levels.find(level => level.label === selectedOption)!;
     const transactionInfo = composedLevels?.[selectedOption];
-    const feeOptions = buildFeeOptions(feeInfo.levels);
+    // Disable custom fees for solana, since the fee is deterministic
```

Comment:
Select bar with one option only is confusing (looks like weird button) and should be replaced by something else or at least hidden.
<img width="804" alt="image" src="https://github.com/trezor/trezor-suite/assets/3729633/115c406a-a059-4057-b67f-d546d630fa9c">